src/server/verification/cron/DisposeEnrollmentsTask.js

```diff
@@ -39,7 +40,7 @@ class DisposeEnrollmentsTask {
     if (exception) {
       const { message: errMessage } = exception
 
-      logger.error(`Couldn't dispose enrollment for ID '${identifier}'`, errMessage, exception, { identifier })
+      logException(logger, `Couldn't dispose enrollment for ID '${identifier}'`, errMessage, exception, { identifier })
```

Comment:
should be error

Reply:
yep here we could do that because here's no additional logic (non-existing case is covered inside and we don't trying to remove if not exists)
so the only case for exception is some unexpected like timeout